fswatch/src/CMakeLists.txt

```diff
@@ -13,22 +13,41 @@
 # You should have received a copy of the GNU General Public License along with
 # this program.  If not, see <http://www.gnu.org/licenses/>.
 #
-set(FSWATCH_SRC_FILES
-        fswatch.cpp
```

Comment:
This applies to many other places: why are we quoting strings when not required?